### PR TITLE
feat(src): add login and register components

### PR DESCRIPTION
**Pull request by jesus-mdz**
add login and register components

### CODE REVIEW

**Review by ahaly92 — APPROVED** [review]

**Review by ahaly92 — APPROVED** [review]

File: src/components/Main/Main.js [hunk 1]
```diff
@@ -15,14 +15,14 @@ function Main() {
...
       <Switch>
-        <Route exact path="/" component={Home}></Route>
-        <Route exact path="/products" component={Products}></Route>
-        <Route exact path="/shopping-cart" component={ShoppingCart}></Route>
-        <Route exact path="/orders" component={Orders}></Route>
-        <Route exact path="/orders-success" component={OrderSuccess}></Route>
-        <Route exact path="/checkout" component={CheckOut}></Route>
-        <Route exact path="/admin-products" component={AdminProducts}></Route>
-        <Route exact path="/admin-orders" component={AdminOrders}></Route>
+        <Route exact path="/" component={Products} />
```
Comment (ahaly92) [inline src/components/Main/Main.js:17]: Nice!

**Review by ahaly92 — APPROVED** [review]

**Comment by mmevans** [comment]
nice work man!

**Review by mmevans — APPROVED** [review]

File: src/components/Form/Form.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  return (
+    <div className="col-lg-4 offset-lg-4">
+      <h2>{title}</h2>
+      <form onSubmit={onSubmit}>
+        {inputArr.map((title, i) => (
+          <Input
+            key={i}
+            title={title}
+            isValid={isValid}
```
Comment (mmevans) [inline src/components/Form/Form.js:30]: Nice work!

**Review by mmevans — APPROVED** [review]

File: src/components/Form/Form.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import Input from '../Input/Input';
+
+function Form(props) {
+  const { title, inputArr, tertiaryButton } = props;
+  const [isValid, setIsValid] = useState('');
+  const [amountTrueArr, setAmountTrueArr] = useState([]);
+
+  const onSubmit = (e) => {
```
Comment (ahaly92) [inline src/components/Form/Form.js]: avoid abbreviations in code. The extra characters are generally worth the readability

File: src/components/Input/Input.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+  useEffect(() => {
+    if (field !== '' && doesCurrentFieldExistOnArray() === false) {
+      setAmountTrueArr([...amountTrueArr, { input: title, validate: true }]);
+    } else if (field === '' && doesCurrentFieldExistOnArray() === true) {
+      setAmountTrueArr(amountTrueArr.filter((obj) => obj.input !== title));
+    }
+  }, [field]);
+
+  const onChange = (e) => {
```
Comment (ahaly92) [inline src/components/Input/Input.js]: same here

File: src/components/Input/Input.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react';
```
Comment (ahaly92) [inline src/components/Input/Input.js]: Since Form and Input are related I would nest in a shared folder

**Review by ahaly92 — APPROVED** [review]

**Pull request merged**